dependencyManagement/build.gradle.kts

```diff
@@ -52,6 +53,8 @@ val DEPENDENCIES = listOf(
   "org.slf4j:slf4j-simple:${slf4jVersion}",
   "org.slf4j:jul-to-slf4j:${slf4jVersion}",
   "io.prometheus:prometheus-metrics-shaded-protobuf:1.3.1",
+  "io.prometheus:prometheus-metrics-exporter-httpserver:${prometheusServerVersion}",
+  "io.prometheus:prometheus-metrics-exposition-formats:${prometheusServerVersion}",
```

Comment:
In `prometheus-metrics-exporter-httpserver:1.3.3`, we got the transitive dependency `prometheus-metrics-exposition-formats` for free. In `1.3.4`, its scope is reduced to a runtime dependency so we need to explicitly add it. See https://github.com/prometheus/client_java/issues/1219 for more details.